<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true">QWidget#centraWidget{
color:rgb(15, 41, 68);
background-color: rgb(255, 255, 255);
}
QToolButton{
font: 20pt &quot;文泉驿等宽微米黑&quot;;
font: 12pt &quot;DejaVu Sans&quot;;
color:rgb(0, 101, 150);
border:2px solid rgb(0, 122, 165);
border-radius:10px;
background-color: rgb(255, 255, 255);
}
QToolButton:disabled{
font: 20pt &quot;文泉驿等宽微米黑&quot;;
font: 12pt &quot;DejaVu Sans&quot;;
color: rgb(180, 180, 180);
border-color: rgb(180, 180, 180);
}
QLabel{
font: 16pt &quot;文泉驿等宽微米黑&quot;;
color:rgb(15, 41, 68);
}</string>
   </property>
   <widget class="QStackedWidget" name="stackedWgtBtn">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1024</width>
      <height>600</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="pageBtn">
     <widget class="QToolButton" name="tBtnConfNode">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>200</y>
        <width>150</width>
        <height>130</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>DejaVu Sans</family>
        <pointsize>12</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="text">
       <string>节点配置</string>
      </property>
      <property name="icon">
       <iconset resource="image.qrc">
        <normaloff>:/Image/node.png</normaloff>:/Image/node.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>80</width>
        <height>80</height>
       </size>
      </property>
      <property name="toolButtonStyle">
       <enum>Qt::ToolButtonTextUnderIcon</enum>
      </property>
     </widget>
     <widget class="QToolButton" name="tBtnConfLoop">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>200</y>
        <width>150</width>
        <height>130</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>DejaVu Sans</family>
        <pointsize>12</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="text">
       <string>回路配置</string>
      </property>
      <property name="icon">
       <iconset resource="image.qrc">
        <normaloff>:/Image/connect.png</normaloff>:/Image/connect.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>80</width>
        <height>80</height>
       </size>
      </property>
      <property name="toolButtonStyle">
       <enum>Qt::ToolButtonTextUnderIcon</enum>
      </property>
     </widget>
     <widget class="QToolButton" name="tBtnConfTime">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>410</y>
        <width>150</width>
        <height>130</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>DejaVu Sans</family>
        <pointsize>12</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="text">
       <string>系统时间</string>
      </property>
      <property name="icon">
       <iconset resource="image.qrc">
        <normaloff>:/Image/time.png</normaloff>:/Image/time.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>80</width>
        <height>80</height>
       </size>
      </property>
      <property name="toolButtonStyle">
       <enum>Qt::ToolButtonTextUnderIcon</enum>
      </property>
     </widget>
     <widget class="QToolButton" name="tBtnConfSystem">
      <property name="geometry">
       <rect>
        <x>830</x>
        <y>200</y>
        <width>150</width>
        <height>130</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>DejaVu Sans</family>
        <pointsize>12</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="text">
       <string>设备配置</string>
      </property>
      <property name="icon">
       <iconset resource="image.qrc">
        <normaloff>:/Image/key.png</normaloff>:/Image/key.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>80</width>
        <height>80</height>
       </size>
      </property>
      <property name="toolButtonStyle">
       <enum>Qt::ToolButtonTextUnderIcon</enum>
      </property>
     </widget>
     <widget class="QToolButton" name="tBtnConfPwd">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>200</y>
        <width>150</width>
        <height>130</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>DejaVu Sans</family>
        <pointsize>12</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="text">
       <string>密码设置</string>
      </property>
      <property name="icon">
       <iconset resource="image.qrc">
        <normaloff>:/Image/pwd.png</normaloff>:/Image/pwd.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>80</width>
        <height>80</height>
       </size>
      </property>
      <property name="toolButtonStyle">
       <enum>Qt::ToolButtonTextUnderIcon</enum>
      </property>
     </widget>
     <widget class="QToolButton" name="tBtnConfNetWork">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>410</y>
        <width>150</width>
        <height>130</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>DejaVu Sans</family>
        <pointsize>12</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="text">
       <string>网络配置</string>
      </property>
      <property name="icon">
       <iconset resource="image.qrc">
        <normaloff>:/Image/selfCheck.png</normaloff>:/Image/selfCheck.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>80</width>
        <height>80</height>
       </size>
      </property>
      <property name="toolButtonStyle">
       <enum>Qt::ToolButtonTextUnderIcon</enum>
      </property>
     </widget>
     <widget class="QToolButton" name="tBtnHostApp">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>830</x>
        <y>410</y>
        <width>150</width>
        <height>130</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>DejaVu Sans</family>
        <pointsize>12</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="text">
       <string>返回程序</string>
      </property>
      <property name="icon">
       <iconset resource="image.qrc">
        <normaloff>:/Image/back.png</normaloff>:/Image/back.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>80</width>
        <height>80</height>
       </size>
      </property>
      <property name="toolButtonStyle">
       <enum>Qt::ToolButtonTextUnderIcon</enum>
      </property>
     </widget>
     <widget class="QToolButton" name="tBtnCalibration">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>410</y>
        <width>150</width>
        <height>130</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>DejaVu Sans</family>
        <pointsize>12</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="text">
       <string>屏幕校准</string>
      </property>
      <property name="icon">
       <iconset resource="image.qrc">
        <normaloff>:/Image/reset.png</normaloff>:/Image/reset.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>80</width>
        <height>80</height>
       </size>
      </property>
      <property name="toolButtonStyle">
       <enum>Qt::ToolButtonTextUnderIcon</enum>
      </property>
     </widget>
     <widget class="QToolButton" name="tBtnNodeAddr">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>200</y>
        <width>150</width>
        <height>130</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>DejaVu Sans</family>
        <pointsize>12</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="text">
       <string>地址管理</string>
      </property>
      <property name="icon">
       <iconset resource="image.qrc">
        <normaloff>:/Image/volume.png</normaloff>:/Image/volume.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>80</width>
        <height>80</height>
       </size>
      </property>
      <property name="toolButtonStyle">
       <enum>Qt::ToolButtonTextUnderIcon</enum>
      </property>
     </widget>
     <widget class="QToolButton" name="tBtnUserLogin">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>410</y>
        <width>150</width>
        <height>130</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>DejaVu Sans</family>
        <pointsize>12</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="text">
       <string>用户登陆</string>
      </property>
      <property name="icon">
       <iconset resource="image.qrc">
        <normaloff>:/Image/login.png</normaloff>:/Image/login.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>80</width>
        <height>80</height>
       </size>
      </property>
      <property name="toolButtonStyle">
       <enum>Qt::ToolButtonTextUnderIcon</enum>
      </property>
     </widget>
     <widget class="QLabel" name="lbBack">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1024</width>
        <height>80</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 30pt &quot;文泉驿等宽微米黑&quot;;
color: rgb(255, 255, 255);
background-color:rgb(0, 122, 165);</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="lbTitle">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>5</y>
        <width>524</width>
        <height>100</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>文泉驿等宽微米黑</family>
        <pointsize>28</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="focusPolicy">
       <enum>Qt::ClickFocus</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 28pt &quot;文泉驿等宽微米黑&quot;;
border:1px solid rgb(0, 122, 165) ;
color: rgb(255, 255, 255);
background-color:rgb(0, 122, 165);
border-bottom-left-radius:40px;
border-bottom-right-radius:40px;
</string>
      </property>
      <property name="text">
       <string> 系统参数设置</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QToolButton" name="tBtnUserLogout">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>410</y>
        <width>150</width>
        <height>130</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>DejaVu Sans</family>
        <pointsize>12</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="text">
       <string>用户注销</string>
      </property>
      <property name="icon">
       <iconset resource="image.qrc">
        <normaloff>:/Image/quit.png</normaloff>:/Image/quit.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>80</width>
        <height>80</height>
       </size>
      </property>
      <property name="toolButtonStyle">
       <enum>Qt::ToolButtonTextUnderIcon</enum>
      </property>
     </widget>
     <zorder>tBtnUserLogout</zorder>
     <zorder>tBtnConfNode</zorder>
     <zorder>tBtnConfLoop</zorder>
     <zorder>tBtnConfTime</zorder>
     <zorder>tBtnConfSystem</zorder>
     <zorder>tBtnConfPwd</zorder>
     <zorder>tBtnConfNetWork</zorder>
     <zorder>tBtnHostApp</zorder>
     <zorder>tBtnCalibration</zorder>
     <zorder>tBtnNodeAddr</zorder>
     <zorder>tBtnUserLogin</zorder>
     <zorder>lbBack</zorder>
     <zorder>lbTitle</zorder>
    </widget>
    <widget class="QWidget" name="pageConf">
     <widget class="QStackedWidget" name="stackedWgtConf">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1024</width>
        <height>600</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QCheckBox{
spacing: 5px;
font: 17pt &quot;文泉驿等宽微米黑&quot;;
color:rgb(15, 41, 68);
border:2px solid rgb(28, 183, 200);
border-radius:10px;
}
QCheckBox::indicator{
border:1px solid black;
width:20px;
height:20px;
margin-left:5px;
border-radius:2px;
image: url(:/Image/uncheck.png);
}
QCheckBox::indicator::unchecked {
image: url(:/Image/uncheck.png);
}
QCheckBox::indicator::checked {
image: url(:/Image/checked.png);
}</string>
      </property>
      <property name="currentIndex">
       <number>-1</number>
      </property>
     </widget>
     <widget class="QToolButton" name="tBtnConfBack">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>885</x>
        <y>530</y>
        <width>120</width>
        <height>60</height>
       </rect>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="text">
       <string>返回</string>
      </property>
      <property name="icon">
       <iconset resource="image.qrc">
        <normaloff>:/Image/back.png</normaloff>:/Image/back.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>35</width>
        <height>35</height>
       </size>
      </property>
      <property name="toolButtonStyle">
       <enum>Qt::ToolButtonTextBesideIcon</enum>
      </property>
      <property name="arrowType">
       <enum>Qt::NoArrow</enum>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="image.qrc"/>
 </resources>
 <connections/>
</ui>
